master.toml

```diff
@@ -1,6 +1,7 @@
 [master]
    version = 1
    runner = "s3://ambiata-dist/master-ambiata/master-scala-oss-0.0.1-20151221033046-cc886a4"
+   sha1 = 13a01b88cc798fa6a89400ab1a77d80adf5dfe32
```

Comment:
It is because it isn't quoted, isn't valid toml, the 13 is a number which would be ok without quotes.